src/components/Link/tests/Link.test.tsx

```diff
@@ -123,4 +123,32 @@ describe('<Link />', () => {
       });
     });
   });
+
+  describe('accessibilityLabel', () => {
+    it('passes prop to the button url is not provided', () => {
+      const mockAccessibilityLabel = 'mock accessibility label';
+      const link = mountWithAppProvider(
```

Comment:
It may be best to use the mountWithApp that has been already imported at the top of the file.